File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.java
class Solution {
    public int[] solution(String s) {
        int zeroCnt=0;
        int loopCnt=0;
        for(; s.length()!=1;loopCnt++){
            int preLen = s.length();
            s= s.replaceAll("0","");
            zeroCnt += preLen- s.length();
            s = Integer.toBinaryString(s.length());
        }
        return new int[]{loopCnt, zeroCnt};
    }
}

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.py
def solution(s):
    remove0=0
    loop=0
    while s!="1":
        preLen=len(s)
        s = s.replace('0','')
        remove0=remove0+preLen-len(s)
        s=str(bin(len(s))[2:])
        loop=loop+1
    return [loop, remove0]